Drift/Private/DriftBase.cpp

```diff
@@ -3359,6 +3359,52 @@ void FDriftBase::RemovePlayerFromMatch(int32 playerID, const FDriftPlayerRemoved
     request->Dispatch();
 }
 
+void FDriftBase::UpdatePlayerInMatch(int32 playerID, const FDriftUpdateMatchPlayerProperties& properties, const FDriftPlayerUpdatedDelegate& delegate)
+{
+    if (state_ != DriftSessionState::Connected)
+    {
+        DRIFT_LOG(Base, Warning, TEXT("Attempting to update player in match without being connected"));
+        delegate.ExecuteIfBound(false);
+        return;
+    }
+
+    DRIFT_LOG(Base, Verbose, TEXT("Updating player '%d' in match '%d'"), playerID, match_info.match_id);
+
+    JsonValue Payload{ rapidjson::kObjectType };
+    if (properties.status.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("status"), properties.status.GetValue());
+    }
+    if (properties.team_id.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("team_id"), properties.team_id.GetValue());
+    }
+    if (properties.statistics.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("statistics"), properties.statistics.GetValue());
+    }
+    if (properties.details.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("details"), properties.details.GetValue());
+    }
+
+    const FString Url = FString::Printf(TEXT("%s/%i"), *match_info.matchplayers_url, playerID);
+    const auto Request = GetGameRequestManager()->Put(Url, Payload);
```

Comment:
Should actually use Patch here. Put is used in some places for legacy reasons when the HTTP library in UE didn't support patch. You'll see that a bunch of APIs in drift-base implement put and patch in the same way, so that patch could be used when it was supported.